src/General.php

```diff
@@ -12,6 +12,57 @@ public function __construct()
         $this->files = $_FILES;
     }
 
+    public function getCssNameFromDb() {
+      return $this->id = $_COOKIE['theme'] ;

```

Comment:
ach, i funkcja  getCssNameFromDb - poza ytm że zrwaca id, to jeszcze przypisuje go do pola id. Śmierdzi ten kawałek. Nazwa nie wskazuje na to ze coś zostanie jednocześnie ustawione....